cpp/test/Ice/logger/log.txt

```diff
@@ -0,0 +1 @@
+-- 05/08/24 16:59:34115 aplicación: info: XXX
```

Comment:
It looks like this file was added by accident.

Reply:
Good catch. I was committing too hastily.
Removed.

Reply:
Maybe we should update the `.gitignore` to catch this file if it was generated.